src/Reflection/ReflectionClass.php

```diff
@@ -874,7 +874,7 @@ public function isInterface() : bool
      * Get the traits used, if any are defined. If this class does not have any
      * defined traits, this will return an empty array.
      *
-     * @return ReflectionClass[]
+     * @return list<ReflectionClass>
```

Comment:
This is discussed here: #530 It raises an error with Psalm that I think is legitimate and I don't know how to handle.